src/app/project/project-details-tab/project-shapefile-resolver.service.ts

```diff
@@ -23,7 +23,7 @@ export class ShapeFileResolver implements Resolve<Observable<object>> {
       currentPage,
       pageSize,
       sortBy,
-      {},
+      { documentSource: 'SHAPEFILE' }, // only look for shapefiles so we are not needlessly parsing through images or other documents
```

Comment:
Comments should start with caps and end with full stops (ie .?!)

Reply:
Fixed!